Description: using shared libuv cannot guarantee io_uring is disabled
 This is CVE-2024-22017
Forwarded: not-needed
Author: Jérémy Lal <kapouer@melix.org>

--- a/test/parallel/parallel.status
+++ b/test/parallel/parallel.status
@@ -12,6 +12,11 @@
 # https://github.com/nodejs/node/issues/52630
 test-error-serdes: PASS, FLAKY
 
+test-process-uid-gid                 : FAIL
+test-process-initgroups              : FAIL
+test-process-setgroups               : FAIL
+test-process-euid-egid               : FAIL
+
 test-process-config                  : PASS,FLAKY
 test-regress-GH-746                  : PASS,FLAKY
 test-stdout-close-unref              : PASS,FLAKY
